source/pymongo-async-migration.txt

```diff
@@ -0,0 +1,357 @@
+
+          await collection.drop()
+   
+   * - ``delete_one()``
+     - .. code-block:: python
+
+          await collection.delete_one(...)
+   
+   * - ``delete_many()``
+     - .. code-block:: python
+
+          await collection.delete_many(...)
+   
+   * - ``find_one()``
+     - .. code-block:: python
+          
+          await collection.find_one(...)
+   
+   * - ``estimated_document_count()``
+     - .. code-block:: python
+
+          await collection.estimated_document_count()
+
+   * - ``count_documents()``
+     - .. code-block:: python
+
+          await collection.count_documents(...)
+
+   * - ``create_index()``
+     - .. code-block:: python
+          
+          await collection.create_index(...)
+
+   * - ``create_indexes()``
+     - .. code-block:: python
+          
+          await collection.create_indexes(...)
+
+   * - ``drop_index()``
+     - .. code-block:: python
+
+          await collection.drop_index(...)
+
+   * - ``drop_indexes()``
+     - .. code-block:: python
+
+          await collection.drop_indexes()
+
+   * - ``list_indexes()``
+     - .. code-block:: python
+
+          await collection.list_indexes()
+
+   * - ``index_information()``
+     - .. code-block:: python
+
+          await collection.index_information()
+
+   * - ``list_search_indexes()``
+     - .. code-block:: python
+
+          await collection.list_search_indexes()
+
+   * - ``create_search_index()``
+     - .. code-block:: python
+
+          await collection.create_search_index(...)
+
+   * - ``create_search_indexes()``
+     - .. code-block:: python
+
+          await collection.create_search_indexes(...)
+
+   * - ``drop_search_index()``
+     - .. code-block:: python
+
+          await collection.drop_search_index(...)
+
+   * - ``update_search_index()``
+     - .. code-block:: python
+
+          await collection.update_search_index(...)
+   
+   * - ``options()``
+     - .. code-block:: python
+
+          await collection.options()
+   
+   * - ``aggregate()``
+     - .. code-block:: python
+
+          async for doc in await collection.aggregate(...):
+             ...
+
+   * - ``aggregate_raw_batches()``
+     - .. code-block:: python
+
+          async for batch in await collection.aggregate_raw_batches(...):
+             ...
+
+   * - ``rename()``
+     - .. code-block:: python
+
+          await collection.rename(...)
+
+   * - ``distinct()``
+     - .. code-block:: python
+
+          await collection.distinct(...)
+
+   * - ``find_one_and_delete()``
+     - .. code-block:: python
+
+          await collection.find_one_and_delete(...)
+
+   * - ``find_one_and_replace()``
+     - .. code-block:: python
+
+          await collection.find_one_and_replace(...)
+
+   * - ``find_one_and_update()``
+     - .. code-block:: python
+
+          await collection.find_one_and_update(...)
+
+Migrate From Motor
+------------------
+
+The {+driver-async+} driver behaves nearly identically to the Motor library. In
```

Comment:
This is a little misleading. Async PyMongo is (or should be) vastly superior to Motor in behavior both in terms of latency and throughput because Async Pymongo directly uses asyncio whereas Motor delegates all work to a thread pool.

Reply:
Updated the wording here, let me know if that works better!